### ISSUE TITLE
Disable the ability to refurbish protected attractions

### ISSUE THREAD

**Issue by Spearkiller** [OpenRCT2/OpenRCT2#13417]
**OS:** Windows 10
**Version:** 0.3.2
**Commit/Build:** d79308e

Just a small thing. On parks like Haunted Harbor, or Sprightly Park, some rides are marked as "protected by local authorities", meaning you can't demolish them or edit their construction. I feel like it's not in the spirit of things to be able to refurbish these rides. A refurbished ride will allow for prices for tickets to be increased and a higher reliability, which removes a lot of the challenge from these scenarios. So, I think it's worth considering, if it hasn't been considered already.

**Comment by pizza2004** [comment]
I think this is a valid comment, but I think it's important to note that once there's a new save format I believe that refurbishment will be reworked to be an actual game mechanic instead of simply resetting the age of the ride, and that many protected and classic rides like this _are_ refurbished regularly to keep them maintained in real life, so a proper rework of refurbishing should fix this point anyway.

**Comment by Gymnasiast** [comment]
I agree with pizza2004 here. Reliability should be based off the last refurbish date, while stuff like maximum ticket price should be based off the actual age.

**Comment by sjoerddebruin** [comment]
Small overlap with #7761.
